lib/csv_importer/runner.rb

```diff
@@ -12,11 +12,26 @@ def self.call(*args)
     attribute :rows, Array[Row]
     attribute :when_invalid, Symbol
     attribute :after_save_blocks, Array[Proc], default: []
+    attribute :batch_size, Integer, default: 1000
+    attribute :use_batches, Boolean, default: false
 
     attribute :report, Report, default: proc { Report.new }
 
     ImportAborted = Class.new(StandardError)
 
+    def initialize(attributes = {})
+      super
+      @csv_reader = CSVReader.new(
+        content: attributes[:content],
+        file: attributes[:file],
+        path: attributes[:path],
+        quote_char: attributes[:quote_char],
+        encoding: attributes[:encoding],
+        batch_size: batch_size,
+        use_batches: use_batches
+      )
+    end
```

Comment:
Instead of overwriting `initialize`, what do you think about extracting `csv_reader` to a method and memoizing it?